### ISSUE TITLE
make creates .c files in the main source folder

### ISSUE THREAD

**Issue by koffie** [mmasdeu/sage_package_template#2]
I just tried using this template and found out that the way this package is building the cython code is quite messy, since it leaves a .c file in the same folder as the pyx file. This is going to be really annoying, since you don't want to add *.c to gitignore, and you also don't want to check which c file you have to manually remove while doing make clean.

**Comment by koffie** [comment]
I did a bit of reading on cython and maybe this is hard to fix since it is the way that cython works (i.e. it doesn't have a directive that allows you to give a "build" directory. The way sage seems to fix this problem, is to first copy over the entire source to a new directory and then build there.

**Comment by mmasdeu** [comment]
Well, let's hope that the projects don't contain a large number of cython files for now...

**Comment by koffie** [comment]
I agree that fixing it right now is maybe not necessary. But I think the issue should not be closed, since it really is an issue which we might want to keep track of.

**Comment by mmasdeu** [comment]
Agreed.
